packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -593,6 +596,28 @@ export interface IFluidDataStoreContext extends IFluidParentContext {
 	 * and its children with the GC details from the previous summary.
 	 */
 	getBaseGCDetails(): Promise<IGarbageCollectionDetailsBase>;
+
+	/**
+	 * Synchronously creates a detached child data store.
+	 *
+	 * The `createChildDataStoreSync` method allows for the synchronous creation of a detached child data store. This is particularly
```

Comment:
Nit: this references `createChildDataStoreSync`, but the method is named `createChildDataStore`

Reply:
```suggestion
	 * The `createChildDataStore` method allows for the synchronous creation of a detached child data store. This is particularly
```